CHANGES.md

```diff
@@ -4,6 +4,8 @@ wavesurfer.js changelog
 4.0.0 (21.06.2020)
 ------------------
 
+- Don't call HTMLMediaElement#load when given peaks and preload == 'none'.
+  Prevents browsers from pre-fetching audio (#1969)
```

Comment:
4.0.0 is already released, you can add a new entry, e.g. 4.1.0.

Reply:
k fixed.